### PR TITLE
IA Guide - Initial Draft

### PR DESCRIPTION
**Pull request by Loquacity**
Issue: https://github.com/thegooddocsproject/planning/issues/12

Epic: https://github.com/thegooddocsproject/planning/issues/2

cc @jaredmorgs

### CODE REVIEW

File: ia-guide/ia-cyoa.tw [hunk 1]
```diff
@@ -215,44 +264,280 @@ And now that they've managed to complete {reader3action2} what is the next thing
...
+reader3action1critscore: critpath19 + critpath20 + critpath21
+reader3action2critscore: critpath22 + critpath23 + critpath24
+reader3action3critscore: critpath25 + critpath26 + critpath27
+--
 
-[[Why do readers read docs?->why]]
+## Writing for your readers
+Now that you've had a think about who your readers are, and the kinds of things they might be using your documentation for, you can display them in a table like this:
+
+<table>
```
Comment (jaredmorgs) [inline ia-guide/ia-cyoa.tw:321]: Do you reckon this would translate well to Markdown or non-interactive mode?

**Comment by mgan59** [comment]
Wanted to be able to review and understand more the approach here with __Twinery__.  I took a shot at running this, let me know if this was the right approach or should I use some other client.  Would be helpful to have more in-depth instructions on how this should run as it will help other maintain this for the future.



Or should I breakup each section into _passages_?  am I using even the correct client?  Let me know if there is a specific client I should be using.

**Comment by Loquacity** [comment]
> Wanted to be able to review and understand more the approach here with **Twinery**. I took a shot at running this, let me know if this was the right approach or should I use some other client. Would be helpful to have more in-depth instructions on how this should run as it will help other maintain this for the future.
> Or should I breakup each section into _passages_? am I using even the correct client? Let me know if there is a specific client I should be using.

This is awesome! You just need to select chapbook as the format, so that the interactive bits work, I think.

File: ia-guide/ia-cyoa.tw (hunk 1 above)
Comment (Loquacity) [reply ia-guide/ia-cyoa.tw:321]: The way I see it, those fields just need to be blank for non-interactive mode. My thinking has been that we need to be able to just remove the interactive elements, and encourage use of a pen and paper.

**Comment by Loquacity** [comment]
Merging this PR now that the initial draft is done. Will open a new PR for editing.

**Pull request merged**